src/main/java/inf112/skeleton/app/Linear.java

```diff
@@ -17,4 +17,12 @@ public static float min(Vector2 v) {
     public static float max(Vector2 v) {
         return Math.max(v.x, v.y);
     }
+
+    /**
+     *
+     * @param v vector to multiply
+     * @param i integer to multiply with
+     * @return scaled copy of given vector
+     */
+    public static Vector2 multiply(Vector2 v, int i) { return new Vector2(v.x * i, v.y * i); }
```

Comment:
Jeg ville ha kalt denne "sca" slik at den har samme navn som i libGDX. Jeg ville også ha brukt float her, slik at metoden både fungerer med int og float.

Reply:
slet med navngiving her ja. sca (scale) impliserer at man scaler gitt vector2, men poenget med å lage en egen var for ikke å scale, men for å returnere en uavhengig vector2

Reply:
Jaja, men de gjør det samme (skalerer en vektor) selv om den ene er statisk, så da trenger man bare å forholde seg til ett navn når man skal skalere en vektor, uavhengig av om man trenger en ny kopi eller om man skal modifisere samme instans.